src/panels/lovelace/common/create-entity-row-element.js

```diff
@@ -0,0 +1,74 @@
+import fireEvent from '../../../common/dom/fire_event.js';
+
+import stateCardType from '../../../common/entity/state_card_type.js';
+import computeDomain from '../../../common/entity/compute_domain.js';
+import { DOMAINS_HIDE_MORE_INFO } from '../../../common/const.js';
+
+import createErrorCardConfig from './create-error-card-config.js';
+
+const CUSTOM_TYPE_PREFIX = 'custom:';
+
+function _createElement(tag, config, stateObj, hass) {
+  const element = document.createElement(tag);
+  try {
+    // eslint-disable-next-line
+    'setConfig' in element && element.setConfig(config);
+  } catch (err) {
+    // eslint-disable-next-line
+    console.error(tag, err);
+    // eslint-disable-next-line
+    return _createErrorElement(err.message, config);
+  }
+
+  const entityId = config.entity;
+  if (entityId && !DOMAINS_HIDE_MORE_INFO.includes(computeDomain(entityId))) {
```

Comment:
I wonder if we should keep this in entities card? Not every place that renders a row might want to trigger a more info?

Reply:
yep https://github.com/home-assistant/home-assistant-polymer/pull/1142

Reply:
This needs to be moved back to `hui-entities-card.js`